File: solutions/lua/darts/1/darts.lua
local Darts = {}

function Darts.score(x, y)
    local distance = math.sqrt(x ^ 2 + y ^ 2)
    if distance > 10 then
        return 0
    end
    if distance > 5 then
        return 1
    end
    if distance > 1 then
        return 5
    end
    return 10
end

return Darts

File: solutions/go/darts/1/darts.go
package darts

func Score(x, y float64) int {
	distanceSquared := x*x + y*y
	if distanceSquared > 100 {
		return 0
	}
	if distanceSquared > 25 {
		return 1
	}
	if distanceSquared > 1 {
		return 5
	}
	return 10
}
